src/System.Management.Automation/engine/SessionStateScopeEnumerator.cs

```diff
@@ -94,7 +95,7 @@ System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator()
 
         public void Dispose()
         {
-            Reset();
+            // Nothing to dispose
```

Comment:
Do we still need the method?

Reply:
Yes, `IEnumerator<T>` implements `IDisposable`.